src/lpdo.jl

```diff
@@ -83,7 +83,7 @@ function normalize!(M::MPO;
   return M
 end
 
-function normalize!(L::LPDO; sqrt_localnorms! = [])
+function normalize!(L::LPDO; sqrt_localnorms! = [], choi::Bool=false)
```

Comment:
I would also prefer a more generic solution here, like generally being able to specify what norm you want to normalize to. For example, we could have a flag `localnorm` which defaults to 1, and then for a Choi matrix or unitary MPO, you can set `localnorm = 2`.

Reply:
Yes, I can add that